src/Symfony/Component/HttpFoundation/HeaderBag.php

```diff
@@ -200,40 +235,65 @@ public function getDate(string $key, ?\DateTimeInterface $default = null): ?\Dat
         return $date;
     }
 
+    /**
+     * Get the default or a targeted cache control instruction set.
+     *
+     * If the set did not exist yet, it is created.
+     */
+    public function getCacheControl(string $target = self::DEFAULT_CACHE_CONTROL_TARGET): CacheControl
+    {
+        // TODO do we need to lowercase the targets as well, and track the desired case as we do for the headers in ResponseHeaderBag
```

Comment:
should we lowercase? and should we preserve original case to recreate the header exactly as desired, even tough its supposed to be case insensitive?